lms/services/lti_outcomes.py

```diff
@@ -233,3 +195,24 @@ def _send_request(cls, outcomes_request_params, pox_body):
             raise LTIOutcomesAPIError("LTI outcome request failed")
 
         return xml
+
+    @classmethod
```

Comment:
I thought for some reason that if you made it static you had to call it by ClassName rather than just self or cls, but that doesn't seem to be the case. Changed that now.